app/src/accounts/accounts.controller.ts

```diff
@@ -8,15 +8,15 @@ import {
 
 import { AccountsService } from "./accounts.service";
 
-export class AccountByHashOrPublicKeyParamDtp {
+export class AccountByHashOrPublicKeyParamDto {
```

Comment:
What is Dtp vs Dto?

Reply:
It was initially a typo. It stands for 'data transfer object'. Essentially what Nestjs calls anything defined to be crossing the wire from client to server

Reply:
Awesome!